packages/shared/src/store/dbHooks.ts

```diff
@@ -353,7 +353,10 @@ export const useJoinedGroupsCount = () => {
 
 export const useGroupByChannel = (channelId: string) => {
   return useQuery({
-    queryKey: [['group', channelId]],
+    queryKey: [
+      ['group', channelId],
+      useKeyFromQueryDeps(db.getGroupByChannel, channelId),
```

Comment:
side note: I know we do this elsewhere, but I'm surprised calling hooks from inside the parameters of another hook is allowed?

Reply:
yeah I know! its just about order of call. as long as you're not conditionally calling it, it's fine to call it however you wish.